app/components/audit_log/Index.js

```diff
@@ -38,11 +38,13 @@ class AuditLogList extends React.PureComponent<Props, State> {
   };
 
   state = {
-    loading: false
+    loading: true
```

Comment:
If we’re setting it on mount, do we need it to be true here?

Reply:
This is a good question, I don't think so actually! Will fix. I think I added it here, aded it below, then forgot to remove it :P